scipy/optimize/_linprog.py

```diff
@@ -270,6 +270,14 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
                 Maximum number of iterations to perform.
             disp : bool
                 Set to ``True`` to print convergence messages.
+            autoscale : bool
+                Set to ``True`` to automatically perform equilibration.
+                Consider this this option the numerical values of
+                constraints are separated by several orders of magnitude.
+            presolve : bool
+                Set to ``False`` to disable automatic presolve.
+            rr : bool
+                Set to ``False`` to disable automatic redundancy removal.
```

Comment:
Uh... Were `presolve` and `rr` never documented?

Reply:
Should the default values be explicitly documented?

Reply:
Done.